relocalization_cfg:
    algorithm_type: "scan_context"
    # algorithm_type: "manually_set"
    lidar_height: 2.2

    # lidar extrinsic to imu
    lidar_ext/x: -0.00665
    lidar_ext/y: 0
    lidar_ext/z: -1.51277
    lidar_ext/roll: 180
    lidar_ext/pitch: 0
    lidar_ext/yaw: 0

bnb3d:
    linear_xy_window_size: 5   # meter
    # linear_xy_window_size: 10   # meter
    linear_z_window_size: 6  # meter
    angular_search_window: 180  # degree 0~180
    pc_resolutions: [0.2, 0.3, 0.5]
    bnb_depth: 3
    bnb_min_score: 0.3

    min_xy_resolution: 0.2
    min_z_resolution: 0.1
    min_angular_resolution: 5   # degree

    thread_num: 6
    filter_size_scan: 0.2
    debug_mode: false

ndt:
    step_size: 0.1
    resolution: 1

gicp:
    gicp_downsample: 0.1
    search_radius: 0.5
    teps: 1.0e-4
    feps: 0.001
    fitness_score: 0.3
